src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,78 @@
+  selectedTodo: Todo,
+  changeSelectedTodo: (el: Todo | null) => void,
+}
+
+export const TodoModal: React.FC<Props> = ({
+  actionClose,
+  selectedTodo,
+  changeSelectedTodo,
+}) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(selectedTodo.userId)
+      .then(res => setSelectedUser(res));
+  }, [selectedTodo]);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+      {selectedUser
+        ? (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${selectedTodo?.id}`}
+              </div>
+
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                type="button"
+                className="delete"
+                data-cy="modal-close"
+                onClick={() => {
+                  actionClose();
+                  changeSelectedTodo(null);
+                }}
+              />
+            </header>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button
-              type="button"
-              className="delete"
-              data-cy="modal-close"
-            />
-          </header>
-
-          <div className="modal-card-body">
-            <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
-            </p>
-
-            <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
-
-              {' by '}
-
-              <a href="mailto:Sincere@april.biz">
-                Leanne Graham
-              </a>
-            </p>
+            <div className="modal-card-body">
+              <p className="block" data-cy="modal-title">
+                {selectedTodo.title}
+              </p>
+
+              <p className="block" data-cy="modal-user">
+                {selectedTodo.completed
+                  ? (
+                    <strong className="has-text-success">Done</strong>
+                  )
+                  : (
+                    <strong className="has-text-danger">Planned</strong>
+                  )}
+
+                {' by '}
+
+                <a href={`mailto:${selectedUser?.email}`}>
+                  {selectedUser?.name}
```

Comment:
same here, you render this jsx only when selectedUser exists, so it looks redundant
```suggestion
                <a href={`mailto:${selectedUser.email}`}>
                  {selectedUser.name}
```